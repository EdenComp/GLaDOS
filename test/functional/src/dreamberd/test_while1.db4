int MIN = 0;
int MAX = 5;
str bl = "
";

fn printLn(s) {
    print(s);
    print(bl);
}

fn main() {
    str s = "Hello World";
    int i = MIN;
    while(i < MAX) {
        print(s);
        printLn(i);
        i = i + 1;
    }
}

main();
