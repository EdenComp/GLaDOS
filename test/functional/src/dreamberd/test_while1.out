Hello World0
Hello World1
Hello World2
Hello World3
Hello World4

